src/pages/ListScholarships.jsx

```diff
@@ -85,6 +101,45 @@ function ListScholarships() {
         <Toolbar />
 
         <Container maxWidth="md" sx={{ flexGrow: 1 }}>
+          <Box
+            sx={{
+              overflowX: 'scroll',
+              whiteSpace: 'nowrap',
+              scrollbarWidth: 'none',
+            }}>
```

Comment:
Hmm, it's not entirely obvious there are more chips, especially with the padding on the right: 
<img width="560" alt="Screen Shot 2022-03-30 at 7 22 33 PM" src="https://user-images.githubusercontent.com/8023233/160963424-5ed9c639-3e59-4c34-bd39-b09e449aa17a.png">

Also the use of screen real estate doesn't look that great on desktop. Kind of makes you wonder why there's so much whitespace on the side.

Couple ideas:
- Add an inset shadow to the edges, kind of like [this one](https://css-tricks.com/books/greatest-css-tricks/scroll-shadows/).
- To maximize screen real estate on large screens what if we place this Box outside of the container and center it? 
  - Alternatively, maybe for `md` screens and up maybe we should show everything no matter how many rows it takes?

(This paragraph is tangential) Something else the use of screen real estate makes me wonder is if we should enlarge the container to `lg` or `xl` and start showing 2 or 3 scholarships per row like we had in our original design 🤔 . It might make better use of space. Along those lines making the filter panel collapsible on bigger screens could be cool too.

Reply:
I kept it inside the container because once I moved it outside, the `overflow` prop would stop working. But I made is so that the tags wrap on `md` screens and higher